src/commons.rs

```diff
@@ -11,8 +11,7 @@ use serde::{Deserialize, Serialize};
 pub const HTTP_TIMEOUT: Duration = Duration::from_secs(30);
 pub const ANSWERING_TASK_INTERVAL_SECONDS: Duration = Duration::from_secs(10);
 pub const FETCH_SPECIFICATION_JSON_MAX_ELAPSED_TIME: Duration = Duration::from_secs(6);
-pub const PIN_CID_LOCALLY_MAX_ELAPSED_TIME: Duration = Duration::from_secs(6);
-pub const PIN_CID_WEB3_STORAGE_MAX_ELAPSED_TIME: Duration = Duration::from_secs(6);
+pub const STORE_CID_LOCALLY_MAX_ELAPSED_TIME: Duration = Duration::from_secs(60);
```

Comment:
I think we can rename this to `STORE_CID_MAX_ELAPSED_TIME` (without locally).

Reply:
Addressed in e694996f0944381a169cb0da2ecb02989470e3b7